sqlitecluster/SQLiteNode.cpp

```diff
@@ -2653,7 +2653,8 @@ void SQLiteNode::postPoll(fd_map& fdm, uint64_t& nextActivity) {
             break;
             case SQLitePeer::PeerPostPollStatus::OK:
             {
-                if (peer->lastActivityTime() && STimeNow() - peer->lastActivityTime() > SQLiteNode::RECV_TIMEOUT - 5 * STIME_US_PER_S) {
+                auto lastSendTime = peer->lastSendTime();
+                if (lastSendTime && STimeNow() - lastSendTime > SQLiteNode::RECV_TIMEOUT - 5 * STIME_US_PER_S) {
```

Comment:
are there any weird "order of operations" in cpp that might lead to the right half of the `&&` not getting executed in the proper order?

Like in PHP with the null coalescing operator we have to make sure we wrap it in `()` when using logical operators `||` and `&&` since something like `if ($myBool || $_REQUEST['mightBeEmpty'] ?? false)` would be computed as `($myBool || $_REQUEST['mightBeEmpty']) ?? false` instead of the intended `$myBool || ($_REQUEST['mightBeEmpty'] ?? false)`